semcore/utils/src/use/useFocusLock.ts

```diff
@@ -166,7 +170,7 @@ const useFocusLockHook = (
         )
           return;
 
-        if (focusCameFrom) {
+        if (focusCameFrom && focusMovedTo) {
```

Comment:
What the point of this condition change?

Reply:
we need this to prevent the focus from remaining on the close button if the next focusable element is inside iframe